ocs_ci/ocs/resources/csi_addons.py

```
@@ -50,17 +53,21 @@ def _restart_csi_addons_controller(namespace: str) -> None:
             break
 
 
-def update_csi_addons_config(key: str, value: str) -> None:
+def update_csi_addons_config(key: str, value: str, restart: bool = True) -> None:
```

Ruff (0.15.13)

[warning] Boolean-typed positional argument in function definition

(FBT001)